shuttleofx_client/views.py

```diff
@@ -3,247 +3,281 @@
 import requests
 from functools import wraps
 from flask import (
-    request,
-    jsonify,
-    render_template,
-    abort,
-    Response,
-    redirect,
-    url_for,
-    session
+	request,
```

Comment:
We need to be really careful about that because it will create a lot of conflicts during merges.